jekyll/_cci2/runner-upgrading-on-server.adoc

```diff
@@ -0,0 +1,116 @@
+---
+version:
+- Server v3.x
+---
+= Upgrading the CircleCI Runner on Server
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+
+This page describes how to update the CircleCI runner on Server.
+
+toc::[]
+
+== Runner for server compatibility
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+| Server version  
+| Launch Agent Version
+
+| 3.0             
+| Runner not supported
+
+| 3.1            
+| 1.0.11147-881b608
+|===
```

Comment:
I think we can take this out of the installation doc and just link to it here maybe?